tests/tests.rs

```diff
@@ -0,0 +1,107 @@
+use multibase::{decode, encode, Base, Base::*};
```

Comment:
Could you please rename the file back to `tests/lib.rs`. I don't see a need to change that.